upgrade/UPGRADE-v9.2.0-dev.md

```diff
@@ -193,3 +193,9 @@ There you can find links to upgrade notes for other versions too.
     - see #project-base-diff to update your project
     - update your dependencies with `composer update` after you set `platform.php` in `composer.json` to the required version
     - if you use custom Dockerfile, don't forget to rebuild your image with the new version of PHP
+- class `\Shopsys\FrameworkBundle\Model\Customer\User\CustomerUser` was changed
```

Comment:
can you please add the PR number in the same manner as it is near other upgrade notes?

Reply:
yes, I also added mention of https://github.com/shopsys/shopsys/pull/2426 for BC break cause by upgrading `roave/better-reflection` few lines above.

Reply:
great 👍